platform/src/main/java/org/stellar/anchor/platform/rpc/NotifyOffchainFundsReceivedHandler.java

```diff
@@ -157,9 +177,12 @@ protected void updateTransactionWithRpcRequest(
       txn.setAmountFee(request.getAmountFee().getAmount());
     }
 
-    JdbcSep24Transaction txn24 = (JdbcSep24Transaction) txn;
-    if (custodyConfig.isCustodyIntegrationEnabled()) {
-      custodyService.createTransaction(txn24);
+    // TODO: add support for SEP-6
```

Comment:
Is it going to be resolved in another PR?

Reply:
Yes, in the PR for ANCHOR-508.